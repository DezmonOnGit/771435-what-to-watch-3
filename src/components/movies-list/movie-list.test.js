import React from "react";
import renderer from "react-test-renderer";
import MoviesList from "./movies-list.jsx";

const movies = [
  {
    name: `test-movie-name-1`,
    img: {
      path: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
      width: 250,
      height: 125
    },
    href: `movie-page.html`
  }
];

describe(`Render movie-list`, () => {

  it(`Render <MovieCard />`, () => {
    const tree = renderer.create(
        <MoviesList
          movies={movies}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
