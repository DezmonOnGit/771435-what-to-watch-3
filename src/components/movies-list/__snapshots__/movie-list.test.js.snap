// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render movie-list Render <MovieCard /> 1`] = `
<div
  className="catalog__movies-list"
>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <div
      className="small-movie-card__image"
    >
      <img
        alt="test-movie-name-1"
        height={125}
        src="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
        width={250}
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        test-movie-name-1
      </a>
    </h3>
  </article>
</div>
`;
